cloudinit/importer.py

```diff
@@ -38,23 +57,16 @@ def find_module(
     """Finds specified modules"""
     if not required_attrs:
         required_attrs = []
-    # NOTE(harlowja): translate the search paths to include the base name.
```

Comment:
a `filter` + `join` accomplishes the same thing, and is less verbose and requires less data manipulation - clean it up while we're here